JavaScript/TypeScript.sublime-syntax

```diff
@@ -202,16 +232,17 @@ contexts:
     - include: ts-namespace-declaration
 
     - match: (?=const{{identifier_break}})
+      pop: true
       branch:
         - ts-const-declaration
         - ts-const-enum
       branch_point: ts-const-enum
 
     - match: (?=declare{{identifier_break}})
```

Comment:
Is there any reason for the absence of `pop: true` here, while it is part of the former one in line 235?